### PR TITLE
Fixed custom action file saving errors

### PR DESCRIPTION
**Pull request by Konti08**
-fix: file format for saving
-enhance: clean up imports of custom actions

### CODE REVIEW

File: action_scripts/convertToLowPoly.py [hunk 1]
```diff
@@ -12,8 +14,8 @@ def convertToLowPoly(parameters, session_id):
        :param parameters: already parsed and checked parameters
        :param session_id: already validated session id of the user
     """
-    image_path = build_image_root_by_id(session_id)
     images = get_from_image_root(session_id)
+    image_format = images[0].split('.')[-1]
```
Comment (Empiire00) [inline action_scripts/convertToLowPoly.py:18]: Image_format -> Image_type?

Comment (Empiire00) [reply action_scripts/convertToLowPoly.py:18]: Every image has same type?

Comment (Konti08) [reply action_scripts/convertToLowPoly.py:18]: According to the definition jpeg and png are image formats. Therefore I think the name image format is suitable.

File: action_scripts/convertToLowPoly.py [hunk 2]
```diff
@@ -22,11 +24,14 @@ def convertToLowPoly(parameters, session_id):
     for file in images:
         t = triangler.Triangler(
             sample_method=triangler.SampleMethod.THRESHOLD, points=250)
```
Comment (Empiire00) [inline action_scripts/convertToLowPoly.py:26]: polygons parameter missing here

File: action_scripts/convertToLowPoly.py (hunk 1 above)
Comment (Konti08) [reply action_scripts/convertToLowPoly.py:18]: > Every image has same type?

Yes all images are from one session and have the same format.

File: action_scripts/convertToLowPoly.py (hunk 2 above)
Comment (Konti08) [reply action_scripts/convertToLowPoly.py]: fixed

**Pull request merged**